tests/unit/test_bruteforce.cpp

```diff
@@ -329,13 +330,108 @@ TEST_F(BruteForceTest, test_bf_info) {
     index = VecSimIndex_New(&params);
     info = VecSimIndex_Info(index);
     ASSERT_EQ(info.algo, VecSimAlgo_BF);
-    ASSERT_EQ(info.d, d);
+    ASSERT_EQ(info.bfInfo.dim, d);
     // User args
     ASSERT_EQ(info.bfInfo.blockSize, 1);
     ASSERT_EQ(info.bfInfo.indexSize, 0);
     VecSimIndex_Free(index);
 }
 
+void compareFlatIndexInfoToIterator(VecSimIndexInfo info, VecSimInfoIterator *infoIter) {
```

Comment:
Consider moving it to `test_utils` (or at least to the top of the file) for making the tests code cleaner..